tests/contrib/operators/test_sql_operator.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from airflow.contrib.operators.sql_operator import SqlOperator
+from airflow import DAG
+from airflow import configuration
+from airflow.utils.tests import skipUnlessImported
+from airflow.utils import timezone
+import unittest
+
+
+class SqlOperatorTest(unittest.TestCase):
+
+    DEFAULT_DATE = timezone.datetime(2015, 1, 1)
+
+    def setUp(self):
+        configuration.load_test_config()
+        args = {'owner': 'airflow', 'start_date': self.DEFAULT_DATE}
+        dag = DAG('unit_test_dag', default_args=args)
+        self.dag = dag
+
+    @skipUnlessImported('airflow.operators.postgres_operator', 'PostgresOperator')
```

Comment:
Can you fix the tests by removing the `skipUnlessImported`? I don't think they are being executed right now.